symfony/src/Service/EnvironmentService.php

```diff
@@ -126,4 +127,47 @@ public function getUserTimezone($useDefault = true)
             return null;
         }
     }
+
+    protected function loadConfiguration($override = [])
```

Comment:
This will need testing with the emulator. I haven't worked a lot with it in Docker.

Reply:
Seems to work as expected on the Test server (after some adjustments in 02168cc).